src/components/learner-credit-management/AssignmentDetailsTableCell.jsx

```diff
@@ -39,6 +40,7 @@ AssignmentDetailsTableCell.propTypes = {
       uuid: PropTypes.string.isRequired,
       learnerEmail: PropTypes.string.isRequired,
       contentKey: PropTypes.string.isRequired,
+      contentTitle: PropTypes.string.isRequired,
```

Comment:
Given `content_title` is currently an [optional field on the model (`null=True, blank=True`)](https://github.com/openedx/enterprise-access/blob/9ad845bfae5ec84028d58bcddbdcf5d98d04048a/enterprise_access/apps/content_assignments/models.py#L145-L152), the prop type may not need the `isRequired` bit.

We also may want to fallback to "View course" when there happens to be no content title as opposed to rendering an empty hyperlink.